in_app_purchases/step_00/app/ios/Podfile

```
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-# platform :ios, '11.0'
+# platform :ios, '12.0'
```

Should be set to 13.0 (uncommented) as required by `cloud_firestore`

```suggestion
platform :ios, '13.0'
```

---

done